rosetta-source/src/main/rosetta/event-position-type.rosetta

```diff
@@ -42,3 +48,35 @@ type Portfolio: <" A Portfolio represents an aggregation of multiple Positions,
 
 	aggregationParameters AggregationParameters (1..1) <"Describes the portfolio by describing how to aggregate all its relevant Events.">
 	portfolioState PortfolioState (1..1) <"Describes the state of the Portfolio as a list of Positions resulting from the aggregation.">
+
+type InventoryBase: <"A data type that can be used to describe an inventory of securities.">
+    inventoryRecord InventoryRecord (0..*) <"An array holding the list of inventory being described. Each element in the inventoryRecord array represents an individual piece of inventory i.e. a security.">
+
+type InventoryRecordBase: <"An individual piece of inventory. This represents a single security.">
+    identifer AssignedIdentifier (1..1) <"Unique identifier for this record. This can be used to uniquely identify a specific piece of inventory.">
+    security Security (1..1) <"The security details.">
```

Comment:
Do you want to make inventory exclusive to `Security` only? Inventory could apply to assets you hold more broadly, so may be more aligned to `Product`. For instance this could apply to `Commodity` as well?

Note: this is related to the broader discussion we're going to have regarding the different layers of "Products".